ckanext/xloader/config_declaration.yaml

```diff
@@ -2,6 +2,23 @@ version: 1
 groups:
   - annotation: ckanext-xloader settings
     options:
+      - key: ckanext.xloader.site_url
+        example: http://ckan-dev:5000
+        default:
+        description: |
+            Provide an alternate site URL for the xloader_submit action.
+            This is useful, for example, when the site is running within a docker network.
+            Note: This setting will not alter path. i.e ckan.root_path
+        validators: configured_default("ckan.site_url",None)
```

Comment:
IMO it's better to just leave it blank by default. It's simpler for code to say "fall back to site_url if the XLoader URL is falsy".

Reply:
May just remove the validators, its not required. (unless you know of one that is testing for url optionally)

Reply:
done